README.md

```diff
@@ -174,13 +174,18 @@ Feel free to check the [issues page](../../issues/).
 
```

Comment:
- [ OPTIONAL] 
Under the `Live Demo` section, you can add the text `Coming Soon...` if you don't yet have a live link.